test/test_jit.py

```diff
@@ -1446,6 +1446,14 @@ def test_fuser_multiple_blocks(this, that, theother, meme):
 
         self.assertEqual(cu.test_fuser_multiple_blocks(*inputs), outputs)
 
+    def test_python_new_trace(self):
+        @torch.jit.trace(torch.zeros(1, 2, 3))
+        def dropout_test(x):
+            x = torch.neg(x)
+            return F.dropout(x)
```

Comment:
This test depends on `Dropout` being implemented in Python. Implement your own autograd function instead, or take the test from my PR.